dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -0,0 +1,450 @@
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.quartz.JobDataMap;
+import org.quartz.JobDetail;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+
+public class SiteSearchJobImplTest extends IntegrationTestBase {
+
+    static ESIndexAPI esIndexAPI;
+    static long defaultLang;
+    static SiteSearchAPI siteSearchAPI;
+    static SiteSearchAuditAPI siteSearchAuditAPI;
+    static ContentletAPI contentletAPI;
+    static HostAPI hostAPI;
+
+    private static String contentGenericId;
+    private static Host site;
+    private static Template template;
+    private static Folder folder;
+    private static User systemUser;
+    private static Container container;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+        LicenseTestUtil.getLicense();
+
+        systemUser = APILocator.systemUser();
+
+        esIndexAPI = APILocator.getESIndexAPI();
+        defaultLang = APILocator.getLanguageAPI().getDefaultLanguage().getId();
+        siteSearchAPI = APILocator.getSiteSearchAPI();
+        siteSearchAuditAPI = APILocator.getSiteSearchAuditAPI();
+        contentletAPI = APILocator.getContentletAPI();
+        hostAPI = APILocator.getHostAPI();
+
+        final List<Host> allHosts = hostAPI.findAll(systemUser, false);
+        for (final Host host : allHosts) {
+            if (host.isSystemHost() || host.getHostname().startsWith("demo")) {
+                continue;
+            }
+            hostAPI.archive(host, systemUser, false);
+            hostAPI.delete(host, systemUser, false);
+        }
+
+        site = new SiteDataGen().nextPersisted();
+        final ContentTypeAPI contentTypeAPI = APILocator.getContentTypeAPI(systemUser);
+        final ContentType contentGenericType = contentTypeAPI.find("webPageContent");
+        contentGenericId = contentGenericType.id();
+
+        final String nameTitle = "myTestContainer" + System.currentTimeMillis();
+        container = new ContainerDataGen()
+                .withContentType(contentGenericType, "$!{body}")
+                .friendlyName(nameTitle)
+                .title(nameTitle)
+                .nextPersisted();
+
+        PublishFactory.publishAsset(container, systemUser, false, false);
+        template = new TemplateDataGen().withContainer(container.getIdentifier()).nextPersisted();
+
+        folder = new FolderDataGen().site(site).nextPersisted();
+
+        PublishFactory.publishAsset(template, systemUser, false, false);
+
+        //Create Contentlet in English
+        final Contentlet contentlet1 = new ContentletDataGen(contentGenericId)
+                .languageId(defaultLang)
+                .folder(folder)
+                .host(site)
+                .setProperty("title", "content1")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "content1" appears 4 times in this file; the first occurrence is on line 121](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093862)